packages/ubuntu_provision/lib/src/accessibility/accessibility_page.dart

```diff
@@ -0,0 +1,159 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:ubuntu_provision/src/accessibility/accessibility_model.dart';
+import 'package:ubuntu_provision/ubuntu_provision.dart';
+import 'package:ubuntu_wizard/ubuntu_wizard.dart';
+import 'package:yaru_icons/yaru_icons.dart';
+import 'package:yaru_widgets/widgets.dart';
+
+class AccessibilityPage extends ConsumerWidget with ProvisioningPage {
+  AccessibilityPage({super.key});
+
+  final ScrollController _scrollController = ScrollController();
+
+  @override
+  Future<bool> load(BuildContext context, WidgetRef ref) async => true;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final model = ref.watch(accessibilityModelProvider);
+    final lang = UbuntuProvisionLocalizations.of(context);
+    final scrollBarPadding =
+        (ScrollbarTheme.of(context).thickness?.resolve({}) ?? 6) * 4;
+
+    return HorizontalPage(
+      windowTitle: lang.accessibilityPageTitle(model.distroName),
+      title: lang.accessibilityPageTitle(model.distroName),
```

Comment:
Could we use the `flavorProvider` instead? Then we wouldn't need the `distroName` property and the `ProductService` in the view model

Reply:
Aah, much better!